tottus/tottus/spiders/tottus_spider.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+import scrapy
+from scrapy import Request
+
+
+class TottusSpiderSpider(scrapy.Spider):
+    name = 'tottus_spider'
+    allowed_domains = ['www.tottus.com.pe']
+    start_urls = ['http://www.tottus.com.pe/tottus/home/']
+
+    def parse(self, response):
+        categories = response.xpath(
+            '//*[@class="nav navbar-nav navbar-categories"]//a[not(@href="#")]/@href').extract()
+        url = 'http://www.tottus.com.pe/tottus/productListFragment'
+        for categorie in categories:
```

Comment:
You should change categorie to category, as categorie doesn't exist see_no_evil, you could accordingly in the file?